app/templates/index.hbs

```diff
@@ -11,16 +11,14 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
-    <div class="upcoming-event">
-    <div class="ui stackable three column grid">
+    <div class="ui stackable three column grid upcoming-event">
```

Comment:
The issue is not in upcoming event section. It's in the event card. Read the issue again